tests/ui/extern-flag/empty-extern-arg.stderr

```diff
@@ -1,13 +1,6 @@
 error: extern location for std does not exist: 
 
-error: `#[panic_handler]` function required, but not found
-
-error: unwinding panics are not supported without std
-   |
-   = help: using nightly cargo, use -Zbuild-std with panic="abort" to avoid unwinding
-   = note: since the core library is usually precompiled with panic="unwind", rebuilding your crate with panic="abort" may not be enough to fix the problem
-
 error: requires `sized` lang_item
 
-error: aborting due to 4 previous errors
+error: aborting due to 2 previous errors
```

Comment:
Was this diagnostics change was expected? (If not, I'll fix it, as I'm working on the regressions caused by this PR)

Reply:
Yes, during my review I assumed this is the weak lang item check that now only looks at crates that are actually used, so we aren't speculatively looking at weak lang items of unused crates anymore

Reply:
From what I see libcore is loaded speculatively in this test (`missing_core` in `fn resolve_crate`) and this error happens due to (weak) lang items in core.